src/scripts/main.js

```diff
@@ -358,3 +358,33 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.getElementsByClassName('dashboard')[0];
+const innerBody = table.lastElementChild;
+
+for (const person of people) {
+  innerBody.appendChild(document.createElement('tr'));
+
+  const line = innerBody.lastElementChild;
+
+  line.appendChild(document.createElement('th'));
+  line.lastElementChild.textContent = person.name;
+  line.appendChild(document.createElement('th'));
+
+  if (person.sex === 'm') {
+    line.lastElementChild.textContent = 'Male';
+  } else if (person.sex === 'f') {
+    line.lastElementChild.textContent = 'Female';
+  }
```

Comment:
Instead of using an if-else statement to determine the gender, you could use a ternary operator. This would make your code more concise.